.github/dependabot.yml

```diff
@@ -15,25 +15,27 @@ updates:
   labels:
     - 'dependencies'
 # maintaining v2
-- package_manager: "java:gradle"
-  target_branch: "release/2.x.x"
+- package-ecosystem: "java:gradle"
```

Comment:
Is "java:gradle" the same like "gradle" used above?

Reply:
[fixed](https://github.com/SDA-SE/sda-dropwizard-commons/compare/9ae8149ac9b36d06162a5c16984f982d7c1d4ea4..cfc18b53e50f76cf31e29dda690120e8440ab273)